### PR TITLE
Refine keys display.

### LINKED ISSUE #7: Show "no keys" message when only revoked keys exist.
When an identity has keys, but they are all revoked, the keys-component table will have a heading but no rows. Probably need the table showing logic to be in the controller and check for non-revoked keys vs just key count over 0.

### PR DESCRIPTION
**Pull request by mattcollier**
Fixes #7.

### CODE REVIEW

File: keys-component.js [hunk 1]
```diff
@@ -69,40 +72,59 @@ function Ctrl($scope, $routeParams, brAlertService, brKeyService,
...
+  function setActiveKeys() {
+    self.activeKeys = false;
+    for(var i = 0; i < self.keys.length; i++) {
+      if(self.keys[i].sysStatus === 'active') {
+        self.activeKeys = true;
+        break;
+      }
+    }
+  }
+
```
Comment (davidlehn) [inline keys-component.js:104]: If we depend on lodash this could be just:
`self.activeKeys = _.some(self.activeKeys, {sysStatus: 'active'})`

Comment (dlongley) [inline keys-component.js:104]: It's not in the dependency list yet (not used anywhere else), so let's just leave it as is for now.

File: keys-component.html [hunk 2]
```diff
@@ -2,21 +2,25 @@
...
+        <a class="stackable-cancel" ng-click="$ctrl.modals.showAddKey=true">
+          <i class="fa fa-plus"></i> Add Key
+        </a>
       </li>
       <li>
-        <a class="stackable-cancel"
-          ng-click="$ctrl.showRevokedKeys=!$ctrl.showRevokedKeys"><i
-          ng-show="$ctrl.showRevokedKeys" class="fa fa-check-square-o"></i><i ng-hide="$ctrl.showRevokedKeys" class="fa fa-square-o"></i> Show Revoked Keys</a>
+        <a class="stackable-cancel" ng-click="$ctrl.showRevokedKeys=!$ctrl.showRevokedKeys">
+          <i class="fa" ng-class="{'fa-check-square-o': $ctrl.showRevokedKeys, 'fa fa-square-o': !$ctrl.showRevokedKeys}">
```
Comment (dlongley) [inline keys-component.html]: Don't need the extra `fa` in the `ng-class` `'fa fa-square-o'` because it's always there via `class="fa"`.

**Comment by dlongley** [comment]
One minor comment to address then LGTM, merge.

**Pull request merged**